tests/unit/amo/components/TestHomeHeroBanner.js

```diff
@@ -1,61 +1,67 @@
 import * as React from 'react';
 import { shallow } from 'enzyme';
 
-import {
-  createFakeEvent,
-  createFakeTracking,
-  fakeI18n,
-  shallowUntilTarget,
-} from 'tests/unit/helpers';
 import HomeHeroBanner, {
+  AB_HOME_HERO_EXPERIMENT,
   AB_HOME_HERO_EXPERIMENT_CATEGORY,
   AB_HOME_HERO_VARIANT_A,
   AB_HOME_HERO_VARIANT_B,
   HomeHeroBannerBase,
 } from 'amo/components/HomeHeroBanner';
 import Hero from 'ui/components/Hero';
+import {
+  createFakeEvent,
+  createFakeTracking,
+  fakeI18n,
+  getFakeConfig,
+  shallowUntilTarget,
+} from 'tests/unit/helpers';
 import { dispatchClientMetadata } from 'tests/unit/amo/helpers';
 
 describe(__filename, () => {
   const defaultProps = {
+    _config: getFakeConfig({
+      experiments: {
+        [AB_HOME_HERO_EXPERIMENT]: true,
+      },
+    }),
     i18n: fakeI18n(),
     store: dispatchClientMetadata().store,
   };
 
-  function shallowRender(props) {
+  function render(props) {
     return shallowUntilTarget(
       <HomeHeroBanner {...defaultProps} {...props} />,
       HomeHeroBannerBase,
     );
   }
 
   it('renders a HomeHeroBanner', () => {
-    const root = shallowRender();
-
+    const root = render();
     expect(root).toHaveClassName('HomeHeroBanner');
   });
 
   it('renders with the "small" experiment classname', () => {
-    const root = shallowRender({ variant: AB_HOME_HERO_VARIANT_A });
+    const root = render({ variant: AB_HOME_HERO_VARIANT_A });
     expect(root).toHaveClassName('HomeHeroBanner--small');
   });
 
   it('renders without the "small" experiment classname', () => {
-    const root = shallowRender({ variant: AB_HOME_HERO_VARIANT_B });
+    const root = render({ variant: AB_HOME_HERO_VARIANT_B });
     expect(root).not.toHaveClassName('HomeHeroBanner--small');
   });
 
   it('renders a carousel with random sections', () => {
-    const root = shallowRender();
+    const root = render();
     const carousel = root.find(Hero);
 
     expect(carousel).toHaveLength(1);
     expect(carousel).toHaveProp('random', true);
   });
 
-  it('sends a tracking event when first rendered on the client', () => {
+  it('sends a tracking event when first render on the client', () => {
```

Comment:
I feel like this sounds a little off maybe..Can we put back rendered or maybe ~ "sends a tracking event *on* first client render" ?